appengine/flexible/tasks/requirements.txt

```diff
@@ -1,3 +1,3 @@
-Flask==1.0.2
-gunicorn==19.9.0
-google-cloud-tasks==0.2.0
+Flask==0.12.2
```

Comment:
Why are we downgrading dependencies here?

Reply:
Merging mistake. Fixed now.